examples/aws-rds/postgresql/rds-postgres-export-blueprint.yaml

```diff
@@ -0,0 +1,148 @@
+          | toString }}'
+        securityGroupID: '{{ .ArtifactsIn.backupInfo.KeyValue.securityGroupID }}'
+        snapshotID: '{{ .ArtifactsIn.backupInfo.KeyValue.snapshotID }}'
+        username: '{{ index .Phases.exportSnapshot.Secrets.dbsecret.Data "username"
+          | toString }}'
+      func: ExportRDSSnapshotToLocation
+      name: exportSnapshot
+      objects:
+        dbsecret:
+          apiVersion: ""
+          group: ""
+          kind: Secret
+          name: '{{ index .Object.data "postgres.secret" }}'
+          namespace: '{{ .Object.metadata.namespace }}'
+          resource: ""
+  restore:
+    inputArtifactNames:
+    - backupInfo
+    kind: ""
+    name: ""
+    phases:
+    - args:
+        instanceID: '{{ .ArtifactsIn.backupInfo.KeyValue.instanceID }}'
+        securityGroupID: '{{ .ArtifactsIn.backupInfo.KeyValue.securityGroupID }}'
+        snapshotID: '{{ .ArtifactsIn.backupInfo.KeyValue.snapshotID }}'
+      func: RestoreRDSSnapshot
+      name: restoreSnapshots
+  restorefromexport:
+    inputArtifactNames:
+    - exportInfo
+    kind: ""
+    name: ""
+    phases:
+    - func: KubeTask
+      name: exportSnapshot
+      objects:
+        dbsecret:
+          apiVersion: ""
+          group: ""
+          kind: Secret
+          name: '{{ index .Object.data "export.secret" }}'
+          namespace: '{{ .Object.metadata.namespace }}'
+          resource: ""
+      args:
+        image: ghcr.io/kanisterio/postgres-kanister-tools:0.78.0
+        namespace: '{{ .Object.metadata.namespace }}'
+        command:
+        - bash
+        - -o
+        - errexit
+        - -o
+        - pipefail
+        - -c
+        - |
+          export PGHOST='{{ index .Object.data "export.host" }}'
+          export PGUSER='{{ index .Phases.exportSnapshot.Secrets.dbsecret.Data "username" | toString }}'
+          export PGPASSWORD='{{ index .Phases.exportSnapshot.Secrets.dbsecret.Data "password" | toString }}'
+          export backup_file_path='test-postgresql-instance/postgres/{{ .ArtifactsIn.exportInfo.KeyValue.backupID}}'
+          kando location pull --profile '{{ toJson .Profile }}' --path ${backup_file_path} - | gunzip -c -f | psql -q -U "${PGUSER}"
+
+
+
+# apiVersion: v1
+# kind: ConfigMap
+# metadata:
+#   name: dbconfig
+#   namespace: rds-app
+# data:
+#   postgres.instanceid:       # instanceid of of the database created in RDS
+#   postgres.host:             # Endpoint of RDS Instance
+#   postgres.databases: |       # databases to take backup of
+#     - test
+#     - test-1
+#   postgres.secret: dbcreds    # name of K8s secret in the same namespace
+#   export.host: # hostname where database needs to be restored
+#   export.port: # port 
+#   export.secret: export-dbcreds
+#   kubectl create secret generic dbcreds -n rds-app --from-literal=username=postgres --from-literal=password=postgres
+#   kubectl create secret generic export-dbcreds -n rds-app --from-literal=username=postgres --from-literal=password=postgres
```

Comment:
How does this fit in with the blueprint? This looks like part of the app, not the blueprint. If it's part of how to set up postgres, it should be in a README.